spaces/S000016/properties/P000151.md

```diff
@@ -0,0 +1,9 @@
+---
+space: S000016
+property: P000151
+value: true
+---
+
+Fix a well-order $\preceq_1$ of all nonempty open subsets of $X$ and a well-order $\preceq_2$ of $X$. In round 0, given the open cover $\mathcal{U}_0$ chosen by Player I, Player II chooses $U_0$ to be the minimum element of $\mathcal{U}_0$ under $\preceq_1$. Then, in round $n$ where $n \geq 1$, given the open cover $\mathcal{U}_n$ chosen by Player I, Player II chooses $U_n$ to be the minimum element (under $\preceq_1$) of $\mathcal{U}_n$ that contains the $n$-th (under $\preceq_2$) element of $X \setminus U_0$, if $X \setminus U_0$ has at least $n$ elements. Player II may choose arbitrarily otherwise.
+
+Since $U_0$ is a nonempty open subset of $X$ and thus cofinite, $\{U_n\}_{n < \omega}$ is always an open cover of $X$.
```

Comment:
```suggestion
A winning strategy $\sigma$ may be defined such that $\sigma(\mathcal U_0)$ is an arbitrary element of $\mathcal U_0$, and we may choose a surjective $x(\mathcal U_0):\omega\to X\setminus\sigma(\mathcal U_0)$. Finally, let $\sigma(\mathcal U_0,\cdots,\mathcal U_n,\mathcal U_{n+1})$ be an arbitrary element of $\mathcal U_{n+1}$ covering $x(\mathcal U_0)(n)$.
```

This is the proof I'd write in a paper. Note that this is really a proof for https://topology.pi-base.org/spaces/S000017/properties/P000151 - we could add this there instead, and then simply assert this as true since it holds for a finer topology. (Another metaproperty we should support someday: mark when spaces are finer/coarser, and when properties automatically hold for finer/coarser topologies...)

Reply:
@StevenClontz Thanks! I was a bit overly cautious mostly since I didn't know the usual level of technical details people write for this kind of proofs. And I was planning to add S17 being P151 as well. I'll just add your proof for S17 being P151 in this PR and assert the trait for S16 as a corollary.

Reply:
@StevenClontz The finer/coarser topologies metaproperty would definitely be extremely useful. I still remember manually adding all sorts of spaces being non-Rothberger just using the topology being finer than a non-Rothberger topology. (Marking a space as another space's Kolmogorov quotient would also be quite useful for similar reasons.)